src/System.Runtime.InteropServices/tests/System/Runtime/InteropServices/RuntimeEnvironmentTest.cs

```diff
@@ -13,9 +13,16 @@ namespace System.Runtime.InteropServices
     public static class RuntimeEnvironmentTest
     {
         [Fact]
-        public static void RuntimeEnvironmentPosTest()
+        public static void RuntimeEnvironmentRuntimeDirectory()
         {
             Assert.True(Directory.Exists(RuntimeEnvironment.GetRuntimeDirectory()));
+        }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot,"RuntimeEnvironmentSysVersion not supported on UapAot")]
```

Comment:
This is returning a version string, not a directory. The implementation just returns `typeof(object).Assembly.ImageRuntimeVersion` so actual problem is that ImageRuntimeVersion is not implemented.

I think RuntimeAssemly.ImageRuntimeVersion should be hardcoded to return `v4.0.30319`. It will make both this API and ImageRuntimeVersion work reasonably well.

The .NET Standard 2.0 is about providing compatible-enough implementations. We are trying to not judge whether the API is "useful".